### DISCUSSION TITLE
What are the steps to publish a plugin?

### DISCUSSION THREAD

**Discussion by alaz-aura** [codeceptjs/CodeceptJS#4132] [Q&A] [answered]
Hi all, I have a plugin that I think would be useful to others and want to know the official to get this from local to npm/yarn/etc. Any guides or steps available?

**Comment by kobenguyent** [comment, marked as answer]
I guess there is no official guidance for this purpose. But you could check the helpers/plugins from community from this link https://github.com/codeceptjs/CodeceptJS/wiki/Community-Helpers-&-Plugins. I believe it would give you some idea on how this works.

**Reply by alaz-aura** [reply]
thanks @kobenguyent , I'll look to imitate other available plugins. Thought having the method to do so formalised in codeceptjs would be better (to avoid guessing all the _right_ ways to publish a plugin)
